crypto/context.c

```diff
@@ -145,6 +146,16 @@ OPENSSL_CTX *OPENSSL_CTX_new(void)
     return ctx;
 }
 
+#ifndef FIPS_MODE
+int OPENSSL_CTX_load_config(OPENSSL_CTX *ctx, const char *config_file)
+{
+    if (!OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CONFIG, NULL))
```

Comment:
Because there are dependancies which we want to ensure are called once before loading..
i.e: openssl_config_int() calls OPENSSL_load_builtin_modules()

Reply:
Note that the way I have changed the code - this call it will only load the default configuration if the environment variable OPENSSL_CONF is set. This means that all existing test code works how it did before without breaking (since OPENSSL_CONF is used by quite a few tests).